values.yaml

```diff
@@ -394,29 +416,22 @@ podMutation:
   #    effect: "NoSchedule"
 
   # Pods spun up would land in the node that matches the affinity
-  affinity: {}
-#    nodeAffinity:
-#      requiredDuringSchedulingIgnoredDuringExecution:
-#        nodeSelectorTerms:
-#          - matchExpressions:
-#              - key: "astronomer.io/dynamic-pods"
-#                operator: In
-#                values:
-#                  - "true"
+  # affinity: {}
+  #   nodeAffinity:
+  #     requiredDuringSchedulingIgnoredDuringExecution:
+  #       nodeSelectorTerms:
+  #       - matchExpressions:
+  #         - key: "company.io/dynamic-pods"
+  #           operator: In
+  #           values:
+  #           - "true"
```

Comment:
Is the top line supposed to be commented out?

Reply:
Ah, good catch. Fixed.